avaframe/ana3AIMEC/aimecTools.py

```diff
@@ -28,18 +28,23 @@
 # -----------------------------------------------------------
 
 
-def readAIMECinputs(avalancheDir, pathDict, dirName='com1DFA'):
+def readAIMECinputs(avalancheDir, pathDict, defineRunoutArea, dirName='com1DFA'):
     """ Read inputs for AIMEC postprocessing
 
     Reads the required geometry files location for AIMEC postpocessing
-    given an avalanche directory; avalanche path, split point and DEM
+    given an avalanche directory; avalanche path, DEM
+    - optional requirement: splitPoint if start of runout area is defined according to startOfRunoutAreaAngle
 
     Parameters
     ----------
+    cfgSetup: configparser object
+        configuration settings of aimec, here used: defineRunoutArea
     avalancheDir : str
         path to directory of avalanche to analyze
     pathDict: dict
         dictionary with info required e.g. reference sim name, comparison type, color variation info
+    defineRunoutArea: bool
+        if True also splitPoint is fetched
```

Comment:
Do we actually need this? Or could we solve it by setting the startOfRunoutAngle to a certain value? I.e. 0 or -999 or soemthing like this?

Reply:
I quite like it, I also think it is not so nice that if set to False, there is a parameter that is not actually used, but this we have more often in the config files and so it is very obvious and it is a feature that is very often used so I like that it is very clear with a flag